packages/sources/coinpaprika/src/config.ts

```diff
@@ -8,9 +8,16 @@ export const DEFAULT_API_ENDPOINT = 'https://api.coinpaprika.com'
 
 export const makeConfig = (prefix?: string): Config => {
   const config = Requester.getDefaultConfig(prefix)
+  const headers: { [T: string]: string | boolean } = {}
+  if (config.apiKey) headers['Authorization'] = config.apiKey
+
+  const isInTestMode = util.getEnv('IS_TEST_MODE', prefix)
+  if (isInTestMode) headers['COINPAPRIKA-API-KEY-VERIFY'] = true
+
```

Comment:
Can you add a TODO/shortcut ticket to remove this once the API is production-ready?

Reply:
Sure.  Created a ticket here https://app.shortcut.com/chainlinklabs/story/25884/remove-is-test-environment-variable-once-coinpaprika-api-goes-live-with-api-keys